packages/common/src/store/playback-position/slice.ts

```diff
@@ -0,0 +1,61 @@
+import { createSlice, PayloadAction } from '@reduxjs/toolkit'
+
+import { ID } from 'models/Identifiers'
+
+import { PlaybackPositionInfo, PlaybackPositionState } from './types'
+
+type SetPlaybackPositionStatePayload = {
+  playbackPositionState: PlaybackPositionState
+}
+
+type SetTrackPositionPayload = {
+  trackId: ID
+  positionInfo: PlaybackPositionInfo
+}
+
+type ClearTrackPositionPayload = {
+  trackId: ID
+}
+
+const initialState: PlaybackPositionState = {
+  trackPositions: {}
+}
+
+const slice = createSlice({
+  name: 'playback-position',
+  initialState,
+  reducers: {
+    // NOTE: This should only be called when seeding the initial state from local storage
+    setPlaybackPositionState: (
```

Comment:
maybe using "initialize" instead of "set" can help invoke this is a one-time deal?